package/android/src/main/java/com/mrousavy/camera/core/CodeScannerFrame.kt

```diff
@@ -0,0 +1,6 @@
+package com.mrousavy.camera.core
+
+data class CodeScannerFrame (
+    val width: Int = 0,
+    val height: Int = 0,
+)
```

Comment:
Nice use of `data class`, but I don't think we need the default value `0` here, do we?